templates/web/fixmystreet.com/index.html

```diff
@@ -0,0 +1,20 @@
+[% map_js = PROCESS 'front/javascript.html' %]
```

Comment:
Move the '<footer>` into the footer-marketing.html templates? And could choose to move the IF pagefooter in there too then, though that's probably not worth it.